sips/sip-012/sip-012-cost-limits-network-upgrade.md

```diff
@@ -526,7 +526,7 @@ Foundation.
 
 * On November 15, 2021, the authors finalized the choice of the activation
   block to be Bitcoin block 713,000.  This block is expected to be mined at or
-around December 6, 2021.
+around December 6, 2021 at 23:00 UTC.
```

Comment:
This change looks fine to me, @wileyj. Can you confirm how you calculated this estimate?

Reply:
Used the current block time average of 8 minutes and 8 seconds, and calculated the diff from current block at a moment in time vs 713000.

resulted in:
561 blocks until 713000
4,532.88 minutes using the 8.08 minute average
75.548 hours using the 8.08 minute average
3.14783333 days using the 8.08 minute average

Then converted the time to UTC from the original point in time from the block diff. 

It's understood the block times will vary, and the estimation is likely going to off.